index.html

```diff
@@ -2094,9 +2095,6 @@ <h4>Resource Categorization Properties</h4>
 							>privacy policy</a>, and <a href="#structural-rel">structural resources</a> like the <a
 							href="#table-of-contents">table of contents</a>.</p>
 
-					<p>Note that a particular resource's <a>URL</a> MUST NOT appear in more than one of these lists, and
-						a URL MUST NOT be repeated within a list.</p>
-
 					<p>The manifest MUST NOT include a reference to itself within any of these lists.</p>
 
```

Comment:
We have to be precise here... If we have a publication that has only one html file in the RO, and the manifest is embedded in that file, then this statement (though precise!) may be misunderstood. The point is that, in such a case, the manifest URL includes a fragment, and it is that _complete_ URL is what MUST NOT be in the RO. The fragment-less URL may.

It may be worth adding a short note to clarify this...

Reply:
Although I sort of wonder why we care if the manifest references itself? It's not like a user agent will get caught in an infinite loop or anything. Maybe a more practical statement might be:

> It is not necessary to include a reference to the manifest in any of these lists.

Just as a contrived example, what if you wrote a guide to publication manifests and wanted to put the manifest json for the publication into the reading order?

Reply:
>  …you wrote a guide to publication manifests and wanted to put the manifest json for the publication into the reading order?

I love this example:-) 

I agree, there is no need to put a "MUST NOT" for something that, for most of the cases, does not make sense anyway (i.e., why would an author do something like that)...

Nuke it!

Reply:
It also means we don't have to have a validation check against every URL in all three lists just to pull out a rather harmless reference.